contents/blog/new-vp-nailing-funnels/new-vp-nailing-funnels.md

```diff
@@ -0,0 +1,80 @@
+
+**Tasks and feature requests**
+
+To gather context on how the team prioritizes and what they’re currently working on, I spent a number of hours reading through open GitHub issues. This gave me insight into what our community is expecting and experiencing when they use PostHog.
+
+### People
+
+**Intro interviews and insights**
+
+I attempted to meet with every single person in the company in my first week - a few meetings spilled over into the second week. I introduced myself and got to learn about them, their history, and what they’re passionate about. I asked everyone the same three questions:    
+
+* In your opinion, what's the long term direction of PostHog?
+* What do you think is the biggest blocker to our success?
+* What can I do to help you become more successful?
+
+The answers to these questions were really enlightening. It was immediately clear that the company had been quite reactive and that a proactive strategy would be welcomed. I also got a great understanding of what the future of PostHog might look like and what I can do to help each individual succeed.
+
+### Insights
+
+Based on these interviews, I aggregated the key themes as a collection of insights to share with the team. This also gave me the opportunity to identify and share a number immediate actions I would be taking to help push the company forward.
+
+## Phase 2: Building the strategy 
+
+### Long Term Direction & Audience
+
+The cornerstone of every strategy is having a long-term direction to point everything else towards. It was easy to build a long term vision for 2023 with insights from the intro interviews and leveraging PostHog’s existing mission. We identified a segment that resonated well with our existing successful customers and has a lot of growth potential.
+
+### First Milestone
+
+With a long term vision, clear focused audience, and a ton of context about our customers, their needs, the market and how our product is used today, we identified one thing we needed to get right before anything else: funnels. 
+
+## Phase 3: Landing the strategy
+
+### Feedback & Iteration
+
+To get buy-in, it was crucial for me to share the draft with the entire company and get as much feedback as possible. The feedback was comprehensive and enabled me to quickly iterate on the first version. To make the strategy easy to remember and execute, we came up with a memorable two-word version of our top priority: “nail funnels” (thanks to [Tim](https://posthog.com/handbook/company/team/marcus-hyett)). 
+
+But you can’t just assume everyone will buy into the strategy because you posted it somewhere and they gave it a thumbs up. You have to talk people through it 1:1 and communicate the message via different channels.. We discussed the new strategy in all our small team standups, company all-hands, and uploaded it to our website.
```

Comment:
Recommend removing the extra period at the end of `communicate the message via different channels..`

Reply:
Thanks for the edits!